src/components/SortSelector.tsx

```diff
@@ -10,13 +10,21 @@ interface SortSelectorOption {
   label: string;
 }
 
-interface SortSelectorProps extends Pick<TextFieldProps, 'sx'> {
+interface SortSelectorProps extends Pick<TextFieldProps, 'sx' | 'variant' | 'size' | 'label'> {
```

Comment:
en siste liten runde med generalisering av props tå tenker jeg denne er god
```suggestion
interface SortSelectorProps extends Pick<TextFieldProps, 'sx' | 'variant' | 'size' | 'label' | 'aria-label'> {
```